data/starts.txt

```diff
@@ -0,0 +1,185 @@
+
+
+
+start "hai space"
+	name "New Hai"
+	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard making human clothing that sold well.`
+	description `	Now, as you enter the shipyard, you feel like this is the start of a new life, in space.`
+	thumbnail "scene/hai start"
+	system "Fah Soom"
+	planet "Greenwater"
+	date 18 6 3020
+	to display
+		or
+			has "main plot completed"
+			has "Ask the Hai about the Unfettered"
+	on display
+		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
+		thumbnail ""
+		date "3020"
+		planet "Slightly too warm"
+		credits "over a million"
+	to reveal
+		has "main plot completed"
+		has "Ask the Hai about the Unfettered"
+	on reveal
+		name "Undiscovered Hai Start"
+		description "There could be interesting possibilities up north, once all crises are resolved."
+		description "	You could have started a different way, went a different path."
+		system "Somewhere far north"
+		planet "Mixed culture"
+		date "middle of 3020"
+		credits 1.431M
+		debt 1.431M
+	to unlock
+		has "main plot completed"
+		has "event: hai-human resolution announced"
+	conversation "hai intro"
+	account
+		credits 1431000
+		score 400
+		mortgage Mortgage
+			principal 1431000
+			interest 0.003
+			term 730
+	set "hai start"
+	# need to set it here to not have it trigger alongside the invisible mission
+	set "First Contact: Hai: offered"
+	set "Intro [0]: offered"
+
+
+
+conversation "hai intro"
+	scene "scene/hai shipyard"
+	`	You've never been to the shipyard before, and going to the poles already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
+	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
+	scene "thumbnail/hai aphid"
+	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`
+	`	You have, after all, heard tales of pirates and quick money from the many newcomers. Thanks to the installation of Mountaintop, they say, the way to human space is much safer than it used to be. What happened with the Pug is not clear, however.`
```

Comment:
"Mountaintop" may need some explanation, I think. Even though the player has played the game before, they might be coming back to it after a while. They may also just not remember what that is.

Reply:
this should no longer be here bc we're not skipping HR